genie-web/src/main/java/com/netflix/genie/web/spring/autoconfigure/events/NotificationsAutoConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ *  Copyright 2019 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.web.spring.autoconfigure.events;
+
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Beans related to external notifications.
+ *
+ * @author mprimi
+ * @since 4.0.0
+ */
+@Configuration
+
```

Comment:
Extra line but why even have this commit when it doesn't contain anything?

Reply:
It creates a new configuration.